utils/transaction-parser.ts

```diff
@@ -289,7 +312,7 @@ function parseZetaInstruction(ix: PartiallyDecodedInstruction): Instruction {
       // size: number;
       let liquidateData = decodedIx.data as zetaTypes.liquidate;
       decodedIx.data = {
-        size: utils.convertNativeLotSizeToDecimal(liquidateData.size)
+        size: utils.convertNativeLotSizeToDecimal(liquidateData.size),
```

Comment:
I think `zetaTypes.liquidate.size` is incorrectly a number.

Reply:
Does the latest commit fix up everything that's necessary for it? I'll try to run a liquidation in devnet later today to make sure the size is passed in correctly

Reply:
I just copied the way that PlaceOrder decodes the size arg